README.md

```diff
@@ -257,3 +143,59 @@ jobs:
     - name: Run `nix build`
       run: nix build .
 ```
+
+
+## Building
```

Comment:
Could we swap "As a Github Action" and "Building", or "As a library" and "Building"? No real reason aside from wanting the like-named sections to be closer together.

Reply:
We can but I think as a github action will be far more common than the other two which are essentially developer only things.